### ISSUE TITLE
events->load() endpoint removes important response data

### ISSUE THREAD

**Issue by christian-ehmig** [checkout/checkout-sdk-php#86]
For events the action load() exists, which loads a single event by ID from the API.
However, the implementation of https://api-reference.checkout.com/#operation/retrieveEvent in the PHP SDK removes the 'data' node of the response which contains the actual data. This renders the load() action useless.

**Comment by rsaestrela** [comment]
Hi @christian-ehmig. We've just updated the codebase and released version 2.0.0-beta3 that contains a new instantiation layer, compatibility with different API Keys, support for OAuth, custom logging & http client and some other interesting features.

The new Events client should be 100% compatible with the API specification and return the data you're looking for. here's a glimpse on how you can get the event details:

```
$event_details = $api->getEventsClient()->retrieveEvent("event_id");
```

To get started, please have a look on the project documentation (README). In the next days, we will upload [extensive docs](https://github.com/checkout/checkout-sdk-php#exception-handling) on how to use each operation. Feel free to create an issue if you have any questions or difficulties migrating to the new version.

**Comment by christian-ehmig** [comment]
Great news, thanks!
